src/flows/pipes/Schedule/remove.ts

```diff
@@ -0,0 +1,14 @@
+import {patchTask} from 'src/client'
```

Comment:
Nit: this should be in the definition, no need for another file

Reply:
updated